docs/spec_draft.md

```diff
@@ -664,16 +671,17 @@ The operation splits the StableHLO grid into process groups as follows:
     `flattened_ids(replica_groups)`.
 
 Afterwards, within each `process_group`:
+
   * `operands@receiver = [operand@sender for sender in process_group]` for all
     `receiver` in `process_group`.
-  * ```
+  *  &nbsp;
```

Comment:
Is this necessary? The current layout has the code block on the same line as the bullet point, the new one has an empty line then the code block (on GH md at least)

Reply:
Same question for all `&nbsp;` introduced in this file.

Reply:
Unfortunately, yes. Python Markdown requires a whitespace after the bullet (so adding the entity makes it explicit, so editors don't remove the trailing space) but PyMd also requires code blocks start on a new line. I couldn't find any extensions to make it work otherwise (but I will send a feature request for the Superfences extension to allow it because I don't see the logic to disallow it). However, I think we should just work-around this shortcoming be avoiding the pattern altogether, because it's grammatically a little weird to start a bullet with a code block; it might be nice to have a phrase that leads into it.

Reply:
Actually I figured out a facepalm-inducing fix for this so the code block appears on the first line: nbsp is a "non-breaking space" that gets parsed by Markdown as a span of text (it gets treated the same as any other character strings), so I replaced it with the unicode for "space" and now the parser ignores it and collapses the code block onto the first line. 👍 

It's still worth considering an introductory phrase for these things, along some other grammatical styles for this document. But now that the visual effect is gone, we can do that as a low-priority cleanup later.

Reply:
I will consider adding one. In the meanwhile thanks for preventing that to become a bottleneck.

Reply:
> It's still worth considering an introductory phrase for these things

Still on board with this. At this point we're relying on parser whitespace inconsistencies. Clever trick to get it to work, nonetheless! Marking as resolved.